locust/passthrough_orderbook_active_orders.py

```diff
@@ -0,0 +1,10 @@
+from locust import HttpUser, task
```

Comment:
Could you please remove the old `orderbook_active_orders.py` test? Seems unnecessary now

Reply:
Removed, good catch! :+1: